socorro/unittest/cron/crontabber_tests_base.py

```diff
@@ -26,20 +24,12 @@
 environment.always_ignore_mismatches = True
 
 
-class TestCaseBase(unittest.TestCase):
-
-    def shortDescription(self):
-        return None
-
-    @classmethod
-    def setUpClass(cls):
-        cls.tempdir = tempfile.mkdtemp()
-
-    @classmethod
-    def tearDownClass(cls):
-        if os.path.isdir(cls.tempdir):
-            shutil.rmtree(cls.tempdir)
```

Comment:
None of the tests use the `tempdir` and the `shortDescription` thing was to fix a really irritating thing that nose did. Since we don't use nose and we're not using the `tempdir`, then we don't need this class at all.